### ISSUE TITLE
go.mod has replace directories causing "go install" to fail

### ISSUE THREAD

**Issue by boojamya** [strangelove-ventures/lens#84]
If a user wants to use Lens inside of another go project, they will need to use `go install github.com/strangelove-ventures/lens@latest`

This command produces the following error:

```
        The go.mod file for the module providing named packages contains one or
        more replace directives. It must not contain directives that would cause
        it to be interpreted differently than if it were the main module.
```

Is it necessary to use these replaces?

https://github.com/strangelove-ventures/lens/blob/69827eda8e577296f191cf7a49f84d22747df4ed/go.mod#L115-L117

**Comment by marbar3778** [comment]
we can remove the grpc replace tag with 0.46 of the sdk. Also I don't think this repo needs the second replace tag, right @fdymylja

**Comment by fdymylja** [comment]
> we can remove the grpc replace tag with 0.46 of the sdk. Also I don't think this repo needs the second replace tag, right @fdymylja

correct. just make sure to upgrade to latest sdk version, otherwise if you use proto or grpc you might incur in panics without the replace directive.
